src/extension/content-script/batteries/Twitter.ts

```diff
@@ -109,10 +109,10 @@ function battery(): void {
         for (const zapElement of zapElements) {
           if (
             (match = (zapElement.nextSibling?.textContent || "").match(
-              /(\S+@\S+)/
+              /\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}\b/
```

Comment:
is a `%` correct?

and the `.` in `[A-Za-z0-9.-]` is probably not correct, do you mean `\.`?

theoretically there are domains with umlauts.
there subdomains and with multiple tlds (e.g. me@bitcoin.me.co.uk) - which your regex matches I think - but I don't know why :D

Reply:
@reneaaron ping

Reply:
Yeah, I didn't consider umlaut domains. Will try to update the regex :+1: